docs/basics/101-125-siblings.rst

```diff
@@ -0,0 +1,190 @@
+
+To get a hang on the basics of sharing a dataset,
+you shared your ``DataLad-101`` dataset with your
+room mate on a common, local file system. Your lucky
+room mate now has your notes and can thus try to catch
+up to still pass the course.
+Moreover, though, he can also integrate all other notes
+or changes you make to your dataset, and stay up to date.
+This is because a DataLad dataset makes updating shared
+data a matter of a single ``datalad update --merge`` command.
+
+But why does this need to be a one-way line? "I want to
+provide helpful information for you as well!", says your
+room mate. "How could you get any insightful notes that
+I make in my dataset, or maybe the results of our upcoming
+mid-term project? Its a bit unfair that I can get your work,
+but you can't get mine."
+
+For example, your room mate might have googled about DataLad
+a bit. On the `datalad homepage <https://www.datalad.org/>`_
+he might have found very useful additional information, such
+as the ascii-cast on `dataset nesting <https://www.datalad.org/for/git-users>`_.
+Because he found this very helpful in understanding dataset
+nesting concepts, he decided to download the ``shell`` script
+that was used to generate this example from Github, and saves
+it in the ``code/`` directory.
+
+Navigate into your dataset copy in ``mock_user/DataLad-101``,
+and run the following command
+
+.. runrecord:: _examples/DL-101-125-101
+   :language: console
+   :workdir: dl-101/DataLad-101
+
+   # navigate into the installed copy
+   $ cd ../mock_user/DataLad-101
+
+   # download the shell script and save it in your code/ directory
+   $ wget -nv https://raw.githubusercontent.com/datalad/datalad.org/7e8e39b1f08d0a54ab521586f27ee918b4441d69/content/asciicast/seamless_nested_repos.sh -O code/nested_repos.sh
+
+Run a quick datalad status:
+
+.. runrecord:: _examples/DL-101-125-102
+   :language: console
+   :workdir: dl-101/mock_user/DataLad-101
+
+   $ datalad status
+
+Save this file:
+
+.. runrecord:: _examples/DL-101-125-103
+   :language: console
+   :workdir: dl-101/mock_user/DataLad-101
+
+   $ datalad save -m "add shell script from ascii cast on nesting" code/nested_repos.sh
+
+Suddenly, your room mate has a file change that you do not have.
+His dataset evolved.
+
+So how do we link back from the copy of the dataset to its
+origin, such that your room mates changes can be included in
+your dataset? How to we let the original dataset "know" about
+this copy your room mate has?
+Do we need to install the installed dataset of our room mate
+as a copy again?
+
+No, luckily, it's simpler and less convoluted. What we have to
+do is to create a datalad ``sibling``: A reference to our room mates
+dataset in our own, original dataset.
+
+.. gitusernote::
+
+   Git repositories can configure clones of a dataset as *remotes* in
+   order to fetch, pull, or push from and to them. A ``datalad sibling``
+   is the equivalent of a git clone that is configured as a remote.
```

Comment:
Rational for using a term other than *remote*: Git operates on *remotes*, git annex adds the concept of *special remotes*. For DataLad I felt that we needed a term that is a superset of both. "special remote" can only be understood, if the meaning/purpose/scope of a "non-special remote" is understood.

In DataLad we want to do nifty things like making a non-git-annex regular remote on github behave like as if it had an annex, by configuring a companion special remote (see https://github.com/datalad/datalad/issues/335, which is the way the studyforrest dataset is distributed).

Having to distinguish these technicalities all the time would be needlessly complex, IMHO.